mcs/class/System/common.sources

```diff
@@ -890,3 +891,11 @@ ReferenceSources/Win32Exception.cs
 
 ../../../external/corefx/src/System.Text.RegularExpressions/src/System/Collections/Generic/ValueListBuilder.Pop.cs
 ../../../external/corefx/src/System.Text.RegularExpressions/src/System/Collections/HashtableExtensions.cs
+
+../../../external/corefx/src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs
```

Comment:
How many entry points we need from this API? We cannot make it public and we don't care about this on mobile profiles